### PR TITLE
Minor additions to the installation guide in README.md.

### PR DESCRIPTION
**Pull request by dsilhavy**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -88,11 +98,15 @@ curl -v http://127.0.0.22:7777/3gpp-m5/v2/service-access-information/d54a1fcc-d4
 }
 ```
 
-The not found response can be tested using a different provisioningSessionId string to the value in the provisioningSessionId key in the configuration YAML file. For example
+The not found response can be tested using a different provisioningSessionId string to the value in the
+provisioningSessionId key in the configuration YAML file. For example
```
Comment (jordijoangimenez) [inline README.md:102]: Just add a : in "For example:"

**Review by jordijoangimenez — CHANGES_REQUESTED** [review]
Just a : missing, the rest looks good!

File: README.md (hunk 1 above)
Comment (dsilhavy) [reply README.md]: Thx done

**Review by jordijoangimenez — APPROVED** [review]

**Review by davidjwbbc — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -1,20 +1,20 @@
...
 
-This repository holds the 5GMS Application Function implementation for the
-5G-MAG Reference Tools.
+This repository holds the 5GMS Application Function implementation for the 5G-MAG Reference Tools.
 
 ## Introduction
 
-The 5GMS application function (AF) is a Network Function that forms part of the
-5G Media Services framework as defined in ETSI TS 126.501.
+The 5GMS application function (AF) is a Network Function that forms part of the 5G Media Services framework as defined
```
Comment (rjb1000) [inline README.md:7]: Should be "5G Media Streaming framework".

**Pull request merged**